common/src/main/java/com/microsoft/identity/common/internal/broker/IntuneMAMEnrollmentIdGateway.java

```diff
@@ -60,34 +63,17 @@
     // time to become compliant and re-request the token.
     private static final long CACHE_TTL_FOR_NULLS_MS = 1500;
 
+    @EqualsAndHashCode
     private static final class CacheKey {
         public final String userId;
         public final String packageName;
 
-        public CacheKey(String userId, String packageName) {
+        public CacheKey(@NonNull final String userId, @NonNull final String packageName) {
             // note: keep values in lower case so string comparisons
             // don't have to be case-insensitive.
             this.userId = userId.toLowerCase();
             this.packageName = packageName.toLowerCase();
         }
-
-        @Override
-        public boolean equals(Object o) {
```

Comment:
Is this because it's unused? if so, we probably can suppress it.

Reply:
No, it was just a syntax issue. If blocks without curly braces. Instead of solving it by adding the curly braces, I decided to rather just delete the method altogether and use Lombok's EqualAndHashCode